apis/mlops/scheduler/scheduler.proto

```diff
@@ -17,6 +17,7 @@ message Model {
   ModelSpec modelSpec = 2;
   DeploymentSpec deploymentSpec = 3;
   StreamSpec streamSpec = 4;
```

Comment:
Just noticed the existence of "StreamSpec" above, holding options related to dataflow (the topic names) which as far as I know are never used. However, let's keep things as they are; we might repurpose the StreamSpec in case we need to configure Core 2 streaming in any way.